test/test.js

```diff
@@ -225,59 +226,67 @@ function noWarnings(input) {
 function missingSourceWarning(input, frequency) {
   var sheet = xlsx.parse(input),
       network = spreadsheetController.parseSheet(sheet);
-  var missingSourceCount = network.warnings.filter(function(x){return x.warningCode=="MISSING_SOURCE"});
+  var missingSourceCount = network.warnings.filter(function(x){return x.warningCode === "MISSING_SOURCE"});
```

Comment:
Note the dangers now of not keeping your peripheral vision open for style adherence opportunities—this block, in some sense, is even worse than before because it uses inconsistent `var` styles within a line of each other.